requirements_all.txt

```diff
@@ -939,6 +939,9 @@ minio==4.0.9
 # homeassistant.components.mitemp_bt
 mitemp_bt==0.0.3
 
+# homeassistant.components.mobilevikings_pl
+mobile-vikings-scraper==1.0.3
```

Comment:
Unfortunately we don't allow integrations that rely on web scraping. So we can't accept this integration with this kind of service interface.

See: https://github.com/home-assistant/architecture/blob/master/adr/0004-webscraping.md